icons/deluge.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Deluge</title><path d="M11.594 23.994a8.462 8.461 0 0 1-1.676-.231 7.861 7.86 0 0 1-2.057-.815l-.24-.138a7.598 7.597 0 0 1-2.339-2.087c-.87-1.17-1.44-2.622-1.63-4.15a8.12 8.12 0 0 1-.069-1.173c0-.41.008-.58.044-.921a8.463 8.462 0 0 1 1.152-3.489c.104-.174.271-.425.378-.569.074-.1 1.698-2.577 5.348-8.15A287.07 287.046 0 0 1 12 0c.003.002.299.451.658 1 4.94 7.556 6.077 9.293 6.187 9.457.344.51.592.964.833 1.53a9.283 9.282 0 0 1 .734 3.39 8.655 8.654 0 0 1-.318 2.605 7.96 7.96 0 0 1-.894 2.03 11.37 11.369 0 0 1-.774 1.074 7.777 7.776 0 0 1-1.159 1.114 8.296 8.295 0 0 1-1.655 1.002 8.93 8.93 0 0 1-4.018.792zm.417-3.119a4.997 4.997 0 0 0 2.48-.804c.177-.118.571-.425.67-.522.096-.094.098-.132.008-.132-.053 0-.206.032-.368.077a8.59 8.59 0 0 1-.506.131c-.546.12-.792.153-1.247.168-.886.03-1.755-.16-2.455-.542a3.793 3.793 0 0 1-1.158-.957 5.364 5.364 0 0 0-.137-.163 4.095 4.095 0 0 1-.888-1.718 4.722 4.722 0 0 1 .045-2.418l.052-.19c.028-.11.095-.306.148-.433a3.875 3.875 0 0 1 1.83-1.97c.48-.242.938-.358 1.497-.379l.148-.005-.072-.049a3.26 3.26 0 0 0-1.217-.496 2.55 2.55 0 0 0-.5-.039 2.734 2.734 0 0 0-.708.08 3.718 3.718 0 0 0-1.195.522 4.745 4.745 0 0 0-.392.296c-.104.089-.385.37-.462.463a3.8 3.8 0 0 0-.529.81l-.119.243a5.569 5.569 0 0 0-.349.874c-.517 1.704-.198 3.615.833 4.982.216.286.445.533.681.736.044.038.11.096.147.131.167.155.396.335.606.473a5.27 5.27 0 0 0 3.157.831zm1.739-2.062c.36-.047.567-.1.935-.24a6.149 6.148 0 0 0 1.768-1.045 9.139 9.138 0 0 0 1.031-1.026c.408-.478.802-1.034 1.208-1.705l.054-.09-.012-.102a7.673 7.672 0 0 0-.574-2.061 7.332 7.331 0 0 0-.679-1.212c-.072-.1-.073-.103-.807-1.228l-1.458-2.23-1.748-2.672-1.157-1.768c-.169-.26-.31-.471-.313-.471-.003 0-1.078 1.637-2.391 3.638a1015.88 1015.794 0 0 1-2.567 3.91c-.185.276-.441.68-.431.68a.795.795 0 0 0 .059-.07c.167-.213.323-.382.499-.543a4.713 4.713 0 0 1 2.489-1.19 3.53 3.53 0 0 1 .564-.037c.258 0 .387.01.601.046.532.087.982.272 1.507.62.641.423 1.228 1.078 1.647 1.842.066.119.221.428.214.428-.003 0-.037-.015-.078-.032a3.259 3.259 0 0 0-.88-.242 2.391 2.39 0 0 0-.399-.017 4.78 4.78 0 0 0-.348.009c-.614.077-1.13.276-1.568.608a2.77 2.77 0 0 0-.673.726c-.031.048-.08.125-.11.169a3.264 3.264 0 0 0-.517 2.257c.054.375.208.779.416 1.093.041.062.101.158.135.214.251.425.633.823 1.063 1.111a3.878 3.878 0 0 0 1.766.637c.153.015.617.011.754-.007z"/></svg>
```

Comment:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Deluge</title><path d="M18.766 10.341 12.006 0l-6.77 10.342c-1.945 2.97-2.191 6.432-.66 9.264C6.04 22.316 8.885 24 12.001 24c3.113 0 5.957-1.681 7.421-4.388 1.532-2.832 1.287-6.297-.657-9.27zm-10.082 6.9c1.433 2.554 3.608 3.045 6.585 2.102-1.7 1.848-5.188 2.337-7.557-.302-1.63-1.817-1.773-4.351-.642-6.468 1.132-2.117 3.388-2.706 5.012-1.551-3.723.09-4.43 4.38-3.398 6.218zm8.72-6.009c.72 1.101 1.163 2.254 1.327 3.4-3.355 5.787-7.873 4.901-9.075 1.395-.554-1.618 1.19-5.08 4.514-3.725-1.296-2.838-4.238-4.017-6.911-1.809a5.037 5.037 0 0 0-.534.546l5.28-8.065 5.398 8.258z"/></svg>
```
This might fix the issue